services/http_gate/cfg/config.yml

```diff
@@ -0,0 +1,20 @@
+logger:
+  level: debug
+  
+rebalance_timer: 5m  # Interval to check nodes health
+
+connect_timeout: 60s  # Timeout to dial node
+
+request_timeout: 300s  # Timeout to check node health during rebalance
+
+# The order in which resolvers are used to find an container id by name
+resolve_order:
+  - nns
+
+server:
+  - address: 0.0.0.0:8080
```

Comment:
Why did we changed the port (`80`-> `8080`)?

Reply:
fixed